### ISSUE TITLE
outdated

### ISSUE THREAD

**Issue by BenjaminHoegh** [Cloudstek/alfred-atom#22]
this project is outdated :(

**Comment by mdeboer** [comment]
I'll update the dependencies today. However I still use it daily and it works fine. It is still up to date with the Alfred API.

Anything you're having trouble with besides the dependencies? Otherwise expect a new release today.

**Comment by mdeboer** [comment]
Small update, a while back I already began rewriting alfred-atom and alfred-hugo in Typescript and updating dependencies. I picked up work again to release this as soon as possible and make it even faster (and up-to date).

It is taking a bit longer than anticipated though as some packages are deprecated without any good alternatives (for me at least). So for example I ended up writing my own simple key/value (cache) store ([@cloudstek/nodejs-cache](https://github.com/Cloudstek/nodejs-cache)) that is used to speed things up.

So 🐻 with me, it's going to be great 👍 Hit _Watch_ above to stay up to date of new releases!
